src/Orc.Controls/Controls/StepBar/Views/StepBarItem.xaml

```diff
@@ -3,25 +3,23 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-             xmlns:xamlbehaviors="http://schemas.microsoft.com/xaml/behaviors"
              xmlns:catel="http://schemas.catelproject.com"
              xmlns:orccontrols="clr-namespace:Orc.Controls"
              mc:Ignorable="d" 
              d:DesignHeight="300" d:DesignWidth="400">
 
-    <xamlbehaviors:Interaction.Triggers>
-        <xamlbehaviors:EventTrigger EventName="MouseDown">
-            <xamlbehaviors:InvokeCommandAction Command="{Binding Command}" 
-                                               CommandParameter="{Binding}" />
-        </xamlbehaviors:EventTrigger>
-    </xamlbehaviors:Interaction.Triggers>
-
-    <Grid ToolTip="{Binding Item.Description}"
-          Cursor="{Binding Item.Command, Converter={orccontrols:CommandCanExecuteToCursorConverter}}">
+    <Grid ToolTip="{Binding Item.Description}">
+        <Button IsEnabled="{Binding Item.Command, Converter={orccontrols:CommandCanExecuteToBooleanConverter}}"
```

Comment:
No, just bind the command. Then the button will be enabled / disabled automatically. So we:

1. Just bind the command to the button
2. Set button visibility to `{Binding RelativeSource={RelativeSource Self}, Path=IsEnabled, Converter={catel:BooleanToCollapsingVisibilityConverter}}`